components/Cargo.toml

```diff
@@ -36,6 +36,7 @@ documentation = "https://eclipse-sdv-blueprints.github.io/blueprints-website/"
 
 [workspace.dependencies]
 async-trait = { version = "0.1" }
+async-std = {version = "1.12", default-features = false }
```

Comment:
do we still need this?

Reply:
async-std removed from Cargo.toml files.